psi4/src/psi4/dfocc/properties.cc

```diff
@@ -46,6 +46,8 @@ void DFOCC::oeprop() {
     // However, CD-OMP2/OLCCD have oeprop enabled but gradients disabled for undocumented reasons.
     // Accordingly, we'll play it safe and not set the density for those cases just yet.
     // ...Which means we can't assume the density to be set in general. Sad. - JPM 07/2020
+    // This makes sense insofar as there are no CD nuclear gradient integrals (there are also no CD-HF gradients)
+    // but you can still calculate one electron properties from the CD-OMP2/OLCCD density - SB 10/2021
```

Comment:
I think that CD gradients are known, see [here](https://onlinelibrary.wiley.com/doi/10.1002/qua.24563) and [here](https://aip-scitation-org.proxy.library.emory.edu/doi/10.1063/5.0087261), but I don't think we have the technology in Psi.

The underlying point that this disabling is probably just because we don't have CD gradient tech is a good one. Propose we change both my comment and Stefan's to the following:
```
    // This density finagling won't be necessary if you just set the density on the wavefunction.
    // We need to set it on CD-OMP2/OLCCD, at least. This should wait until after the current dfocc push. - JPM 08/2022
```

Reply:
ok, after reading the code conversation and your proposed replacement comment, I proffer the below as my summary understanding.

```
    // This density finagling won't be necessary if you just set the density on the wavefunction.
    // We do need to set density explicitly on CD-OMP2/OLCCD, at least, since one-electron properties
    // accessible for CD but CD gradients (which would set the density automatically) are not. (That
    // is, CD gradient tech available in the literature but none in Psi4, even for CD-HF.)
    // This should wait until after the current dfocc push. - JPM 08/2022
```